Tools/cases_generator/generators_common.py

```diff
@@ -75,47 +111,73 @@ def __init__(self, out: CWriter):
             "ERROR_IF": self.error_if,
             "ERROR_NO_POP": self.error_no_pop,
             "DECREF_INPUTS": self.decref_inputs,
+            "DEAD": self.kill,
+            "INPUTS_DEAD": self.kill_inputs,
```

Comment:
I feel that `KILL` and `KILL_INPUTS` are better names than `DEAD` and `INPUTS_DEAD` (and they would also mirror the function  names as for the other macros).

Reply:
I prefer `DEAD` over `KILL`, as it states that the variable is dead at this point. It doesn't instruct the code generator to somehow kill it.

"killing" is what happens during the analysis. The live variable becomes dead.
In bytecodes.c, we are declaring that the variable is dead at this point.